nwbinspector/config.schema.json

```
@@ -0,0 +1,23 @@
+{
+  "type": "object",
+  "properties": {
+    "CRITICAL": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "BEST_PRACTICE_VIOLATIONS": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "BEST_PRACTICE_SUGGESTIONS": {
```

```suggestion
    "BEST_PRACTICE_VIOLATION": {
      "type": "array",
      "items": {
        "type": "string"
      }
    },
    "BEST_PRACTICE_SUGGESTION": {
```

These should exactly match the name of the corresponding `Importance` levels, right?